src/Seo.php

```diff
@@ -166,4 +166,53 @@ public static function getProductBrand($data) {
     return $data;
   }
 
+  /**
+   * Fixes schema.org prices according to tax settings.
+   *
+   * When retrieving product prices to add into schema.org woocommerce is not
+   * considering if they already includes taxes, as set in the backend. This
+   * seems to be caused by aelya-currency-switcher currency conversion.
+   *
+   * See https://github.com/woocommerce/woocommerce/blob/bfac625cdcda4d7d14e85ca3cf5534e86081e7bc/includes/class-wc-structured-data.php#L222-L223
```

Comment:
Perhaps we could shorten this URL with Bitly or some similar tools?
At least this is what I do in such cases.